modules/indexer/issues/internal/model.go

```diff
@@ -26,7 +26,7 @@ type IndexerData struct {
 	LabelIDs           []int64            `json:"label_ids"`
 	NoLabel            bool               `json:"no_label"` // True if LabelIDs is empty
 	MilestoneID        int64              `json:"milestone_id"`
-	ProjectID          int64              `json:"project_id"`
+	ProjectIDs         []int64            `json:"project_id"`
```

Comment:
```suggestion
	ProjectIDs         []int64            `json:"project_ids"`
```
Other places also need to be modified.